### DISCUSSION TITLE
Threads Android Demo

### DISCUSSION THREAD

**Discussion by Antoinegtir** [Antoinegtir/instagram-threads#21] [Announcements]
Stable diffusion that showcase the advancement state of the Threads Clone

<hr /><em>This discussion was created from the release <a href='https://github.com/Antoinegtir/instagram-threads/releases/tag/0.0.2'>Threads Android Demo</a>.</em>

**Comment by spalheapp** [comment]
The application did not work, it had a gray screen and there was no registration area as seen in Print.

**Reply by Antoinegtir** [reply]
this one should work I fix the issue 👀 <a href="https://github.com/Antoinegtir/instagram-threads/releases/tag/0.0.3">Click here</a>

**Comment by spalheapp** [comment]
It worked, I had some problems, the screen is still grey, some functions like username and links don't work and when following a user it doesn't follow and the follower's profile has an edit button.  Other than that congratulations beautiful project.

**Reply by Antoinegtir** [reply]
Thank you don't worry I will solve all the problem this is just the beginning

**Reply by spalheapp** [reply]
Congratulations 😁

**Reply by spalheapp** [reply]
hey bro new update?

**Reply by Antoinegtir** [reply]
soon i finish the function that allow to create post and then I will do the UI & list the posts in feed and inside the profile

**Reply by spalheapp** [reply]
Hi bro can you generate the new apk for me to test

**Reply by Antoinegtir** [reply]
Yes

**Reply by spalheapp** [reply]
As soon as you update the new app you let me know so I can download it. ☺️

**Reply by Antoinegtir** [reply]
yes ;)

**Reply by Antoinegtir** [reply]
New release of Android version @spalheapp <a href="https://github.com/Antoinegtir/instagram-threads/releases/tag/0.0.5">Click here</a>

**Reply by spalheapp** [reply]
hi bro the new app the screen is grey.  I downloaded the new update.

**Reply by Antoinegtir** [reply]
I test my new release on every devices on android that I have (emulated or not) there is no grey screen since a long time Idk why you are the only guy that got this bug I think that you should retry on another android device or clean cache of your emulator because I have no idea sorry

**Reply by spalheapp** [reply]
Strange, I deleted the cache, installed it 3x and it doesn't show me the Feed or the profile.

**Reply by Antoinegtir** [reply]
> Strange, I deleted the cache, installed it 3x and it doesn't show me the Feed or the profile.

Yes really strange and did you clone my project, run it in order to start debugging mode and check where is the breakpoint where data value is null

**Reply by spalheapp** [reply]
Ok. Thank you Man. 😊
